📝(doc) improve README (#80)
openfun/cunningham: README.md

> <div align="center">
>   <a href="https://openfun.github.io/cunningham"><img src="./resources/banner.png" alt="" /></a>
> </div>
> <br/>
> <p align="center">
>     <a href="https://openfun.github.io/cunningham"><b>📚&nbsp;&nbsp;Documentation</b></a> •
>     <a href="https://www.figma.com/file/sSEMhow3g5zpEkyDLEQ1ZO/DS-Cunningham?node-id=47%3A261&t=nKRnadK4sslyVxD7-0"><b>🖌️&nbsp;&nbsp;Figma</b></a>
> </p>

Reviewer: I'll just keep a level-one header especially for SEO.

```suggestion
<div align="center">

<a href="https://openfun.github.io/cunningham"><img src="./resources/banner.png" alt="" /></a>

# Cunningham

**A design system and a components library.**

<p>
    <a href="https://openfun.github.io/cunningham"><b>📚&nbsp;&nbsp;Documentation</b></a> •
    <a href="https://www.figma.com/file/sSEMhow3g5zpEkyDLEQ1ZO/DS-Cunningham?node-id=47%3A261&t=nKRnadK4sslyVxD7-0"><b>🖌️&nbsp;&nbsp;Figma</b></a>
</p>

</div>
```

WDYT ?
<img width="757" alt="image" src="https://github.com/openfun/cunningham/assets/9265241/1dc614d6-29fc-45b8-b650-36a4b699c84a">

See the [documentation](https://openfun.github.io/cunningham) 📚 to learn more about customization, typography, spacings and available components!

</div>

## ⚡️ Getting started

➡️ First, install the library

```
yarn add @openfun/cunningham-react
```

➡️ Then, add this script to your `package.json` file
```json
{
    "scripts": {
        "build-theme": "cunningham -g css -o src"
    },
}
```

➡️ Then, in order to generate the css file, run

```
yarn build-theme
```

It will generate a file named `cunningham-tokens.css`.

➡️ Then, add these lines at the top of your main stylesheet file:

```
@import "@openfun/cunningham-react/fonts"; // Imports default fonts ( Roboto ). You can also import fonts by yourself.
@import "@openfun/cunningham-react/icons"; // Imports icons ( Material Icons ).
@import "@openfun/cunningham-react/style"; // Imports the default theme.
@import "cunningham-tokens"; // Imports the file you just generated.
```

> Please take a look at [customization documentation](https://openfun.github.io/cunningham/?path=/docs/getting-started-customization--docs) for more informations about the `cunningham` command line tool and to customize your local theme.

We're done! 🎉

> Now to verify that everything works well by trying to render a component, like the [Button](https://openfun.github.io/cunningham/?path=/docs/components-button--docs) for example.

Reviewer: ```suggestion
To ensure everything works well, let's test rendering a component, such as the [Button](https://openfun.github.io/cunningham/?path=/docs/components-button--docs).
```

Now please see the [documentation](https://openfun.github.io/cunningham) 📚 to learn more about customization, typography, spacings and available components!

## Contributors

<a href="https://github.com/openfun/cunningham/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=openfun/cunningham" />
</a>

## Contributing

> 

Reviewer: In other repository we also have those sentences.

```
This project is intended to be community-driven, so please, do not hesitate to get in touch if you have any question related to our implementation or design decisions.

We try to raise our code quality standards and expect contributors to follow the recommandations from our [handbook](https://openfun.gitbooks.io/handbook/content).
```
